web/src/server/api/routers/traces.ts

```diff
@@ -577,6 +581,7 @@ function createTracesQuery(
             scores
         WHERE
             trace_id = t.id
+            AND scores."data_type" IN ('NUMERIC', 'BOOLEAN')
```

Comment:
why exclude categorical?

Reply:
Only used for traces filter logic, scores are fetched via separate query